### PR TITLE
add UNICORN bid adapter

### PR DESCRIPTION
**Pull request by ctylim**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- Add UNICORN bid adapter
- PR for Documentation: https://github.com/prebid/prebid.github.io/pull/1827

- test parameters for validating bids
```
{
  bidder: 'unicorn',
  params: {
    placementId: 'rectangle-ad-1',
    bidfloorCpm: 0.2,
    accountId: 12345,
    bcat: ['IAB-1', 'IAB-2']
  }
}
```

### CODE REVIEW

File: modules/unicornBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  utils.logInfo('[UNICORN] OpenRTB Formatted Request:', request);
+  return JSON.stringify(request);
+}
+
+const interpretResponse = (serverResponse, request) => {
+  utils.logInfo('[UNICORN] interpretResponse.serverResponse:', serverResponse);
+  utils.logInfo('[UNICORN] interpretResponse.request:', request);
+  const res = serverResponse.body;
+  var bids = []
+  res.seatbid.forEach(sb => {
```
Comment (musikele) [inline modules/unicornBidAdapter.js]: during my tests, `res` was `""` (empty string) and this caused `res.seatbid` to fail. This must be fixed.

**Comment by ctylim** [comment]
Hi @musikele,
Thank you for your review, fixed some errors.

**Comment by musikele** [comment]
@ctylim the PR looks good to me 
you only need to rebase this to latest `master` commit, because prebid.js introduced a series of new checks and test fixes on CI. Once you do this, I'll merge. 
If you need any help let me know 
Thank you !

**Comment by ctylim** [comment]
@musikele I'm done!

**Pull request merged**